src/test/regress/expected/local_shard_utility_command_execution.out

```diff
@@ -0,0 +1,637 @@
+NOTICE:  truncate cascades to table "dist_table_1500001"
+NOTICE:  truncate cascades to table "dist_table_1500004"
+NOTICE:  truncate cascades to table "dist_table_1500007"
+NOTICE:  truncate cascades to table "dist_table_1500010"
+NOTICE:  truncate cascades to table "dist_table_1500013"
+NOTICE:  truncate cascades to table "dist_table_1500016"
+NOTICE:  truncate cascades to table "dist_table_1500019"
+NOTICE:  truncate cascades to table "dist_table_1500022"
+NOTICE:  truncate cascades to table "dist_table_1500025"
+NOTICE:  truncate cascades to table "dist_table_1500028"
+NOTICE:  truncate cascades to table "dist_table_1500031"
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500001 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500004 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500007 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500010 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500013 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500016 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500019 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500022 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500025 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500028 CASCADE
+NOTICE:  executing the command locally: TRUNCATE TABLE local_commands_test_schema.dist_table_1500031 CASCADE
+-- show that TRUNCATE is successfull
+SELECT COUNT(*) FROM ref_table, dist_table;
+ count
+---------------------------------------------------------------------
+     0
+(1 row)
+
+---------------------------------------------------------------------
+------ local execution of DROP ------
+---------------------------------------------------------------------
+-- droping just the referenced table would error out as dist_table references it
+DROP TABLE ref_table;
+ERROR:  cannot drop table ref_table because other objects depend on it
+DETAIL:  constraint fkey on table dist_table depends on table ref_table
+HINT:  Use DROP ... CASCADE to drop the dependent objects too.
+-- drop those two tables via remote execution
+DROP TABLE ref_table, dist_table;
+-- drop the other standalone table locally
+DROP TABLE ref_table_1;
+NOTICE:  executing the command locally: DROP TABLE IF EXISTS local_commands_test_schema.ref_table_1_1500033 CASCADE
+CONTEXT:  SQL statement "SELECT master_drop_all_shards(v_obj.objid, v_obj.schema_name, v_obj.object_name)"
```

Comment:
consider  using

```SQL
\set VERBOSITY terse
```

to prevent CONTEXT messages

Reply:
thanks, I didn't even know this